src/views/partials/card.handlebars

```diff
@@ -0,0 +1,52 @@
+{{#each card}}
+<div class="card-modal" hidden>
+    <button class="close-button" id="closeModal">Close Modal</button>
+    <form id="updateCard" action="/board/card/{{boardId}}/{{_id}}" method="POST">
```

Comment:
using id in a loop is gonna make multiple things with the same id

Reply:
your route should only ever be giving me 1 card, this is just so I dont have to type card._id, card.members, card.name, etc.

Reply:
okay dope